masu/processor/orchestrator.py

```diff
@@ -35,16 +35,40 @@ class Orchestrator():
 
     """
 
-    def __init__(self):
+    def __init__(self, billing_source=None):
         """
         Orchestrator for processing.
 
         Args:
-            None
+            billing_source (String): Individual account to retrieve.
         """
-        self._accounts = AccountsAccessor().get_accounts()
+        self._accounts = self.get_accounts(billing_source)
```

Comment:
Isn't this a static method?  If so, it should be called like this: `Orchestrator.get_accounts()`.

This looks like you're calling an instance method.